gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 accompanist = "0.28.0"
 androidDesugarJdkLibs = "2.0.3"
-androidGradlePlugin = "8.1.1"
+androidGradlePlugin = "8.2.0-alpha14"
```

Comment:
Updating to alpha versions of AGP is pretty risky IMO.

Reply:
Required for FTL https://cs.android.com/android-studio/platform/tools/base/+/mirror-goog-studio-main:firebase/testlab/testlab-gradle-plugin/src/main/java/com/android/tools/firebase/testlab/gradle/TestLabGradlePlugin.kt;l=52

Reply:
Yes, I understand that this version is a prerequisite for enabling FTL managed devices. I'm simply pointing out that it might have unexpected consequences on the overall build, since AGP is applied on almost all modules of this project. 😃